python/run-tests.py

```diff
@@ -99,8 +99,60 @@ def run_python_style_checks(root_dir):
     run_cmd([os.path.join(root_dir, "dev", "lint-python")], stream_output=True)
 
 
+def run_pypi_packaging_tests(root_dir):
```

Comment:
add methods docs explaining what this is method is doing. its a little non-trivial